tests/README.md

```diff
@@ -2,6 +2,17 @@
 
 This project contains Renku acceptance tests.
 
+There are several acceptance tests, which are all in the ch.renku.acceptancetests package.
+
+| Class Name                | Purpose                                                          |
+| ------------------------- | ---------------------------------------------------------------- |
+| LoginSpec                 | Just check that login/logout works                               |
+| MinimalFunctionalitySpec  | Check that the basic features are working                        |
+| HandsOnSpec               | Run through the hands-on tutorial                                |
+| ExistingProjectSpec       | Not normally run, but useful for debugging/developing            |
+
+Some tests create new projects (MinimalFunctionalitySpec, HandsOnSpec). In these cases, the project name is normally of the form `test_[timestamp]`. This makes it easy to identify the test projects and generally avoid name collisions. If the test runs successfully to completion, the project is deleted at the end of the test. If the test fails, then the project is left around to aid debugging.
```

Comment:
could you please wrap this at some reasonable character count?

Reply:
It's markdown. What is the advantage of hard wrapping the content?

Reply:
it's a text file. single-line text files are annoying to read.